<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="Qurd" failures="0" tests="1" name="Default test" time="0.177" errors="0" timestamp="2025-05-23T08:01:41 EDT">
  <testcase classname="com.qa.automationexercise.base.BaseTest" name="@BeforeTest setup" time="0.029">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.util.Properties.getProperty(String)&amp;quot; because &amp;quot;prop&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.util.Properties.getProperty(String)" because "prop" is null
at com.qa.automationexercise.factory.DriverFactory.initDriver(DriverFactory.java:38)
at com.qa.automationexercise.base.BaseTest.setup(BaseTest.java:25)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:657)
at org.testng.TestRunner.beforeRun(TestRunner.java:646)
at org.testng.TestRunner.run(TestRunner.java:612)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- @BeforeTest setup -->
  <testcase classname="com.qa.automationexercise.base.BaseTest" name="@AfterTest tearDown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterTest tearDown -->
  <testcase classname="com.qa.automationexercise.tests.LoginPageTest" name="logoExistTest" time="0.0">
    <skipped/>
  </testcase> <!-- logoExistTest -->
  <testcase classname="com.qa.automationexercise.tests.LoginPageTest" name="loginPageURLTest" time="0.0">
    <ignored/>
  </testcase> <!-- loginPageURLTest -->
  <testcase classname="com.qa.automationexercise.tests.LoginPageTest" name="loginTest" time="0.0">
    <ignored/>
  </testcase> <!-- loginTest -->
  <testcase classname="com.qa.automationexercise.tests.LoginPageTest" name="loginPageTitleTest" time="0.0">
    <ignored/>
  </testcase> <!-- loginPageTitleTest -->
</testsuite> <!-- Default test -->
